install_config/install/deploy_router.adoc

```diff
@@ -265,6 +265,193 @@ $ iptables -A INPUT -p tcp --dport 10443 -j ACCEPT
 ----
 ====
 
+
+[[creating-router-shards]]
+=== Creating Router Shards
+
+Router sharding is
+link:../../architecture/core_concepts/routes.html#router-sharding[based on labels];
+you set labels on the routes in the pool,
+and express the desired subset of those routes for the router to serve
+with a selection expression either when creating the router,
+or at a later time with the `oc edit` command.
+
+The rest of this section describes an extended example.
+Suppose there are 26 routes, named `a` -- `z`,
+in the pool, with various labels:
+
+.Possible labels on routes in the pool
+----
+sla=high       geo=east     hw=modest     dept=finance
+sla=medium     geo=west     hw=strong     dept=dev
+sla=low                                   dept=ops
+----
+
+These labels express the concepts:
+service level agreement, geographical location,
+hardware requirements, and department.
+The routes in the pool can have at most one label from each column.
+Some routes may have other labels, entirely, or none at all.
+
+[options="header",cols="1,1,1,1,1,3"]
+|===
+|Name(s) |SLA |Geo |HW |Dept |Other Labels
+
+|`a`
+|`high`
+|`east`
+|`modest`
+|`finance`
+|`type=static`
+
+|`b`
+|
+|`west`
+|`strong`
+|
+|`type=dynamic`
+
+|`c`, `d`, `e`
+|`low`
+|
+|`modest`
+|
+|`type=static`
+
+|`g` -- `k`
+|`medium`
+|
+|`strong`
+|`dev`
+|
+
+|`l` -- `s`
+|`high`
+|
+|`modest`
+|`ops`
+|
+
+|`t` -- `z`
+|
+|`west`
+|
+|
+|`type=dynamic`
+
+|===
+
+Here is a convenience script *_mkshard_*  that
+centralizes the common arguments of the `oadm router` command:
+
+====
+[source,bash]
+----
+#!/bin/bash
+# Usage: mkshard ID SELECTION-EXPRESSION
+id=$1
+sel="$2"
+oadm router router-shard-$id \//<1>
+    --selector="$sel" \//<2>
+    --replicas=3 \
+    --credentials=/etc/origin/master/openshift-router.kubeconfig \
+    --service-account=ipfailover
```

Comment:
You don't need to specify `--credentials` and `--service-account` unless you want to override the defaults. I'd just drop those from the example here.